Write better description of room and FOSDEM value proposition (#4)
pietroppeter/fosdem2022: README.md

**Elaborate description of proposed devroom (including possible topics)**:

> Nim is a versatile language that can both be compiled to low-level languages

Reviewer: ```suggestion
Nim is a versatile language that can be both compiled to low-level languages
```

like C/C++/ObjC and to the web-oriented language JavaScript. This, along with

> it's rich, static type system, tuneable and optional GC, and its rich

Reviewer 2: *tunable* is the more widespread variant spelling. Perhaps, "optional" should be put first.

> meta-programming system the language offers something for everyone. Being used

Reviewer: ```suggestion
meta-programming system, the language offers something for everyone. Being used
```

unique combination of features and the aforementioned meta-programming makes it
feel at home in pretty much every programming niche.

Looking back at the talks FOSDEM has had about Nim in the past we expect a
varied selection of talks ranging from the design of the language and compiler
itself by core developers all the way to the many aforementioned use-cases for
such a language. Previous FOSDEM talks:

- [Nim on everything, from microcontrollers to web-sites, C and JS as intermediary languages](https://archive.fosdem.org/2020/schedule/event/nimoneverything/)
- [Move semantics in Nim, Deterministic Memory Management](https://archive.fosdem.org/2020/schedule/event/nimmovesemantics/)
- [Designing an ultra low-overhead multithreading runtime for Nim, Exposing fine-grained parallelism for 32+ cores hardware via message passing](https://archive.fosdem.org/2020/schedule/event/nimultralowoverheadruntime/)
- [Async await in Nim, A demonstration of the flexibility metaprogramming can bring to a language](https://archive.fosdem.org/2020/schedule/event/asyncawaitnim/)
- [Metaprogramming with Nim](https://archive.fosdem.org/2019/schedule/event/nim_metaprogramming/)

> - [How to build your own Ethereum client, The development story of Nimbus](https://archive.fosdem.org/2019/schedule/event/nimbus/)

Reviewer: Epic list!

Author: I know right! Hope I haven't forgotten any :thinking:

**Why does it fit FOSDEM?**

The Nim programming language first saw the light of day in 2008, and has been

> in active development ever since. In more recent years it has reached stable

Reviewer: ```suggestion
developed actively as an open source project ever since. In more recent years it has reached stable
```

> versions with v1.0 in 2019, and the latest version v1.6.0 released late 2021.

Reviewer: ```suggestion
the milestone 1.0 release in 2019, with the latest version v1.6.0 released late 2021.
```

The language has been open source since it's very beginning, and is being

> developed by a group of volounteers along with a few developers paid by

Reviewer 2: *volunteers*
"a few" > "a core team of" ?

donations from the community. The Nim community itself has traditionally been
very Euro-centric, with most of the core developers and users from Europe.

> FOSDEM has previouvsly been an annual meeting ground for Nim users where many

Reviewer 2: *previously*

have been able to put a face to the nicknames we interact with every day for

> the first time. With this following there is also the possibility to transition

Reviewer: "With this following..." needs to be rewritten

I guess you mean: "Following this there is also...."

Author: That's not what I meant at all. I was refering to the European following, as in the people who follows Nim.

Author: I went with "userbase", should make the sentence a bit more clear.

> into a future physical room in the future once FOSDEM is back to being a

Reviewer 2: One future is enough.

physical event.

The Nim community has also organised its own conference, NimConf2020 and

> NimConf2021, and the people behind this proposal have already helped out with

Reviewer 2: Perhaps mention some numbers? Total talks, views?

Author: I was trying to not talk too much about NimConf in case it detracts from our proposal that we already have our conference. Maybe they want to give a voice to some smaller community. Not sure though.

Besides, I didn't have any numbers around, and as I said this is all written in one go :P
